bazel/envoy_mobile_dependencies.bzl

```diff
@@ -23,6 +23,7 @@ def swift_dependencies():
 def kotlin_dependencies():
     maven_install(
         artifacts = [
+            "com.google.code.findbugs:jsr305:3.0.2",
```

Comment:
i didn't find where this library is used in the code, could you point it for me 😬

Reply:
It's used to provide the `javax.annotation.Nullable` annotation to let me mark a Java function as being able to accept a Kotlin nullable in the jvm-only code (so no android Nullable)

I couldn't find any other Nullable annotations so I went with this, lmk if there is a better way to go about this :)